### DISCUSSION TITLE
Add a plugin in a pre-compiled binaries Sofa

### DISCUSSION THREAD

**Discussion by Dong-Le-gnoD** [sofa-framework/sofa#4658] [Write / Run a simulation] [answered]
Hi,
I am using a pre-compiled binaries 23.06 Sofa. Even though there are several available plugins, I need the plugin SofaViscoElastic. How do I add the plugin and build the Sofa?
Thank you!

**Comment by alxbilger** [comment, marked as answer]
Hi,

you can find some details here: https://www.sofa-framework.org/community/doc/plugins/build-a-plugin-from-sources/. In your case, if you keep the SOFA binaries, you should go with the "out-of-tree" compilation.
